docs/pages/versions/unversioned/sdk/light-sensor.md

```diff
@@ -0,0 +1,170 @@
+---
+title: LightSensor
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-sensors'
+packageName: 'expo-sensors'
```

Comment:
Please do replace all the occurrences of `Barometer` with `Light Sensor` and update the descriptions accordingly. I marked only a few of them, but there are more.

Reply:
I have now!